content/slides/data-formats-and-inspection.md

```diff
@@ -3,8 +3,6 @@ title: "Data/metadata inspection"
 subtitle: "QGreenland Researcher Workshop 2023"
 index: 30
 background-image: "https://live.staticflickr.com/65535/50238782702_020e861875_k.jpg"
-title-slide-attributes:
-  data-background-image: "https://live.staticflickr.com/65535/50238782702_020e861875_k.jpg"
```

Comment:
These lines were useful when we were using local files as backgrounds. Simply using `background-image` would not copy the local file into the bundle and slides would get a 404 trying to find the background. Adding the title slide attribute would ensure the file was correctly copied. Now that we're using remote background images, we don't need to copy files into the bundle and can remove the hack.